### PR TITLE
Turn on strict concurrency.

### PR DESCRIPTION
**Pull request by mbrandonw**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -27,35 +27,38 @@ let package = Package(
         "PerceptionMacros",
         .product(name: "OrderedCollections", package: "swift-collections"),
         .product(name: "XCTestDynamicOverlay", package: "xctest-dynamic-overlay"),
+      ],
+      swiftSettings: [
+        .enableExperimentalFeature("StrictConcurrency")
```
Comment (stephencelis) [inline Package.swift:32]: Let's tag this branch with a pseudo-release so that we can confirm that this doesn't break downstream builds.

**Review by stephencelis — APPROVED** [review]

**Comment by Brett-Best** [comment]
Might be worth turning on [GlobalConcurrency](https://github.com/apple/swift-evolution/blob/main/proposals/0412-strict-concurrency-for-global-variables.md) too?

**Comment by mbrandonw** [comment]
Hi @Brett-Best, thanks for the suggestion! I can definitely add that, but it also seems to be a subset of StrictConcurrency as far as I can tell. I get the global mutable state warning when only StrictConcurrency is specified in Package.swift.

Are you aware of anything that GlobalConcurrency surfaces that is not covered by StrictConcurrency?

**Comment by Brett-Best** [comment]
Ah, looks like it's enabled implicitly when StrictConcurrency is on 😅
https://github.com/apple/swift/blob/17ca88c94a34b34c3c354891e899e82ce98f46ee/lib/Frontend/CompilerInvocation.cpp#L979-L983

**Pull request merged**